std_msgs/Header header
float64 size
string session_id